src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -178,7 +178,18 @@ def to_content_str_or_list(chat_str: str, hash2images: Mapping):
     return result if include_image else chat_str
 
 
-def handle_openai_error(tries: int = 10, delay: float = 8.0):
+def generate_retry_interval(retry_count: int) -> float:
+    minBackoffInSec = 3
+    maxBackoffInSec = 60
+    retry_interval = minBackoffInSec + ((2 ** retry_count) - 1)
```

Comment:
Just curious why minus 1?

Reply:
For the first retry, we want it to use the min_backoff_in_sec (3s) directly. And here the retry_count is 0, so use 2**retry_count - 1 to achieve " + 0".